CHANGELOG.md

```diff
@@ -40,6 +40,7 @@
     - `-experimental.tsdb.stripe-size` changed to `-experimental.blocks-storage.tsdb.stripe-size`
     - `-experimental.tsdb.wal-compression-enabled` changed to `-experimental.blocks-storage.tsdb.wal-compression-enabled`
     - `-experimental.tsdb.flush-blocks-on-shutdown` changed to `-experimental.blocks-storage.tsdb.flush-blocks-on-shutdown`
+* [CHANGE] Clarify limitations to the `/series` endpoint. #2953
```

Comment:
Any documentation update should be an `[ENHANCEMENT]`. Changes are application-level behavioural changes.

Reply:
Moved to [ENHANCEMENT]